src/components/Stake.svelte

```diff
@@ -94,38 +91,40 @@
   }
 </script>
 
-<div class:hidden={false} class="action-card basis-1/2">
-  <p class="action-card-title">
-    Stake to Provider Id {providerId}
-  </p>
-  <div>
-    <ol class="ml-4 mt-4 list-decimal font-light">
-      <li>Ensure the control key has a FRQCY balance.</li>
-      <li>Click 'Stake'</li>
-      <li>This will require 1 signature to send the transaction.</li>
-    </ol>
-  </div>
-  <form class="mt-8">
-    <KeySelection
-      component="SelectControlKey"
-      selectLabel="Key to Stake From"
-      bind:selectedOption={selectedKey}
-      {validAccounts}
-      classOverrides="border-2 rounded-lg"
-    />
-    <div class="mt-6">
-      <label for="stakingInput">Amount to Stake in <span class="units">{token}</span></label>
-      <div class="input-container mt-2">
-        <input type="number" id="stakingInput" value="1" on:input={handleInput} />
+<Modal id="stake-to-provider" {isOpen} {close}>
+  <BlockSection title="Stake to Provider">
+    <form class="column w-[320px]">
+      <DropDownMenu
+        id="stake-using-key"
+        label="Wallet Address"
+        bind:selected={selectedAccount}
+        options={Array.from($providerAccountsStore.values())}
+        formatter={formatAccount}
+        onChange={() => {}}
+      />
+      <div class="mt-6">
+        <label for="stakingInput">Amount in <span class="units">{token}</span></label>
```

Comment:
```suggestion
        <label for="stakingInput" class="label">Amount in <span class="units">{token}</span></label>
```

Reply:
^^ Make sure same styling as dropdown menu label & select

Reply:
Since the tag is `label`, should we just make all labels have a class that includes `label` in the main CSS file?

Reply:
> @claireolmstead wrote:
> ^^ Make sure same styling as dropdown menu label & select

As in  `class="label mb-3.5 block"` ?  yes?